django/db/models/functions/date.py

```diff
@@ -0,0 +1,214 @@
+from datetime import datetime
```

Comment:
Renaming this file to `datetime.py` and adding `from __future__ import absolute_import` at the top to prevent collisions with the built-in module on Python 2 is working for me. Did you see/consider that suggestion? I do think that name is more intuitive.